lib/git_reflow/git_server/pull_request.rb

```diff
@@ -156,54 +158,56 @@ def commit_message_for_merge
           message << "\nLGTM given by: @#{lgtm_authors.join(', @')}\n"
         end
 
-        message
+        "#{message}\n"
       end
 
       def cleanup_feature_branch?
-        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to cleanup your feature branch? ") =~ /^y/i
+        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to push this branch to your remote repo and cleanup your feature branch? ") =~ /^y/i
       end
```

Comment:
I removed this in my previous commit PR because I thought that this message could be possibly be a bit misleading.

It is true that we always push our feature branch, although I was a bit concerned about this message because we don't always push the `base_branch` to the remote (in the case of `deliver` using the `github_api` gem, we don't push the `base_branch` because a user can add an extra commit for whatever reason to `base_branch` between `review` and `deliver.`